README.md

```diff
@@ -2,83 +2,40 @@
-the code that is deployed is often called a "Service", to avoid confusion, this
-document will refer to the code that accesses a service as an "application".
-
-**Application operator**: the person or team responsible for deploying an
-application. Users in this role, at minimum, have access to their own
-application's namespace. In some cases, users in this role may also be an
-application developer or a cluster operator
-
-**Cluster operator**: the person or team responsible for operating a Kubernetes
-cluster. This team may operate the cluster on behalf of other users, or may
-operate the cluster to facilitate their own work
-
-**Catalog operator**: the person or team responsible for adminstration of the
-Service Catalog, including catalog curation and Service Broker registration
-
-**Broker operator**: the person or team responsible for running and managing one
-or more **Service Brokers**.
-
-**Service Producer**: the person or team who authors and/or operates a Service
-available from the Service Catalog. As part of creating a service, the Service
-Producer may also be running a Service Broker.
-
-**Resource type**: a logical Kubernetes concept. Examples include:
-
-  - [Pod](http://kubernetes.io/docs/user-guide/pods/)s
-  - [Service](http://kubernetes.io/docs/user-guide/services/)s
-  - [Secret](http://kubernetes.io/docs/user-guide/secrets/)s
-
-**Resource**: a specific instantiation of an aforementioned resource type,
-often represented as a YAML or JSON file that is submitted or retrieved via the
-standard Kubernetes API (or via `kubectl`)
-
-**Binding**: represents a relationship between an Application and a Service
-Instance. A Binding contains the information necessary for the Application to
-make use of the Service Instance.
-
-### Purpose and Scope
+## Contributing
 
-The exact purpose of the SIG will grow over time, and as such, please see
-the various "version" folders under the [docs](./docs) directory for the
-exact list of use-cases, features and design decisions that have been agreed to.
+Interested in contributing?  Check out the [documentation](./CONTRIBUTING.md)
 
-However, this SIG is (initially) focused on implementing integration with the
-[Open Service Broker API](https://github.com/openservicebrokerapi/servicebroker).
-Additionally, we will be exploring what can be done to help Service developers
-when they want to expose their code via a Service Broker.
+## Kubernetes Incubator
 
-We are currently scoping our **v1** milestone [here](./docs/v1). Interested
-in contributing?  Check out the [documentation](./CONTRIBUTING.md)
+This is a [Kubernetes Incubator project](https://github.com/kubernetes/community/blob/master/incubator.md).
+The project was established 2016-Sept-12.  The incubator team for the project is:
 
-## Contributing
+- Sponsor: Brian Grant (@bgrant0607)
+- Champion: Paul Morie (@pmorie)
+- SIG: [sig-service-catalog](https://github.com/kubernetes/community/tree/master/sig-service-catalog)
```

Comment:
@duglin The information belongs in kubernetes/community -- we can put any info we want there.